eox_nelp/notifications/tests/test_tasks.py

```diff
@@ -374,3 +375,23 @@ def _generate_subsections(self, subsections_data):
             subsections.append(subsection)
 
         return subsections
+
+
+class NotifyUpcomingCourseDueDateByIdTestCase(unittest.TestCase):
```

Comment:
the task could raise `DoesNotExist` exception, should we handle that ?

Reply:
https://github.com/eduNEXT/eox-nelp/pull/44/commits/4299f3197f74f8f14c154d002844a00beb73e4f9#diff-80232d77731fe3d63292ad0abfb7fece44750d6e59f2573efefc2864703b1afbR369